integration/spark-common/src/main/scala/org/apache/carbondata/spark/util/CommonUtil.scala

```diff
@@ -288,6 +297,69 @@ object CommonUtil {
     result
   }
 
+  def validateForOverLappingRangeValues(desType: Option[String],
+      rangeInfoArray: Array[String]): Boolean = {
```

Comment:
@QiangCai ....Please correct me if I am wrong. Scala has already a predefined method for comparing different array elements. I think for writing a Comparator class we will have to write our own logic which will be an extra overhead to maintain.